docs/openapi.yaml

```diff
@@ -59,6 +59,11 @@ paths:
             format: uuid
           required: true
           description: UUID identifier of the Publication to fetch.
+        - in: query
+          name: doNotRedirect
+          schema:
+            type: boolean
+          description: If fetching a deleted / unpublished publication, the response code will be 410, instead of 301. A non deleted / non-unpublished will respond as if the query param was not set.
```

Comment:
Hvis denne er `true` så returneres 410 i stedet for 301? Så klienten kan på en måte overstyre http-koden APIet egentlig ønsker å sette?

Reply:
Jepp, designpris. Men jeg fant ingen andre fornuftige måter å gjøre dette på.

Når PO ønsker i normal situasjon at brukeren bli redigert, men i særtilfeller ønsker å få se det sletta greiene, så måtte vi la klienten få styre hva den skal få tilbake.